### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by martindekov**
Syncing up the documentation in a couple of places because
of the feedback of user in slack also forgot to update it
with the limits change

Signed-off-by: Martin Dekov (VMware) <mdekov@vmware.com>

## Description

User in slack gave feedback on couple of places where documentation is misleading or he had troubles so I fixed that. Also adds info on PR #164 which I forgot to document

## How Has This Been Tested?

N.A.

## How are existing users impacted? What migration steps/scripts do we need?

Their experience deploying OF-cloud will be easier

## Checklist:

I have:

- [x] updated the documentation and/or roadmap (if required)
- [x] read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] signed-off my commits with `git commit -s`
- [ ] added unit tests

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ You will create/deploy:
 * A GitHub App
 * A number of secrets
 * A stack of OpenFaaS functions via stack.yml
-* Customise limits for Swarm or K8s
```
Comment (alexellis) [inline docs/README.md]: The British spelling is valid.

**Comment by alexellis** [comment]
HI @martindekov I merged @rgee0 first, so please can you fix up if we still need any of your edits?

File: docs/README.md (hunk 1 above)
Comment (martindekov) [reply docs/README.md]: I hope you are not offended it just gave me green underlining ..

**Comment by martindekov** [comment]
I will take a look at his PR and remove the unnecessary parts

**Comment by alexellis** [comment]
Looks fine :+1:

**Pull request merged**